shared/react-native/android/app/src/main/java/io/keybase/ossifrage/MainActivity.java

```diff
@@ -68,12 +69,16 @@ protected void onCreate(Bundle savedInstanceState) {
 
         super.onCreate(savedInstanceState);
 
-        if (getIntent().getExtras().containsKey("notification")) {
-            ReactContext currentContext = getReactInstanceManager().getCurrentReactContext();
-            if (currentContext != null) {
-                currentContext
-                        .getJSModule(DeviceEventManagerModule.RCTDeviceEventEmitter.class)
-                        .emit("androidIntentNotification", "");
+        Intent intent = getIntent();
+        if (intent != null) {
+            Bundle bundle = intent.getExtras();
+            if (bundle != null && bundle.containsKey("notification")) {
+                ReactContext currentContext = getReactInstanceManager().getCurrentReactContext();
+                if (currentContext != null) {
+                    currentContext
```

Comment:
worth checking the return value of `getJSModule`?

Reply:
mmmm couldn't hurt. this plus instancemanager i think can't be null ever